analyzer/src/main/kotlin/managers/Npm.kt

```diff
@@ -196,6 +196,7 @@ open class Npm(
         }
     }
 
+    private val artifactoryApiPathPattern = Regex("(.*artifactory.*)(?:/api/npm/)(.*)")
```

Comment:
What do you think about adding `.*jfrog.*` as an alternative match?

Reply:
Do we have a real-life example for that?

Reply:
No.